pytorch_lightning/profiler/simple.py

```diff
@@ -72,13 +72,21 @@ def stop(self, action_name: str) -> None:
         duration = end_time - start_time
         self.recorded_durations[action_name].append(duration)
 
-    def _make_report(self) -> Tuple[list, float]:
+    def _make_report_extended(self) -> Tuple[List, float, float]:
         total_duration = time.monotonic() - self.start_time
-        report = [[a, d, 100.0 * np.sum(d) / total_duration] for a, d in self.recorded_durations.items()]
-        report.sort(key=lambda x: x[2], reverse=True)
-        total_calls = sum(len(x[1]) for x in report)
+        report = [
+            [a, np.mean(d), len(d), np.sum(d), 100.0 * np.sum(d) / total_duration]
+            for a, d in self.recorded_durations.items()
+        ]
+        report.sort(key=lambda x: x[4], reverse=True)
+        total_calls = sum(x[2] for x in report)
         return report, total_calls, total_duration
 
+    def _make_report_non_extended(self) -> List[Tuple[str, float, float]]:
```

Comment:
nit: if one is `*_extended` this can be just `_make_report`

```suggestion
    def _make_report(self) -> List[Tuple[str, float, float]]:
```